Source/Core/pages/knowMore.html

```diff
@@ -158,7 +159,7 @@ <h2 class="section-heading">Learn More about Carbon Footprint</h2>
                                         </select>
                                     </div>
                                 </form>
-                                <span class="pull-right"><i>Source: </i><a href="http://aqicn.org/">http://aqicn.org/</a></span>
+                                <span class="pull-right"><i>Source: </i><a href="http://aqicn.org/" style="color:#110401">http://aqicn.org/</a></span>
```

Comment:
That is not an appropriate way of changing the colour of all links. You must change the colour in the appropriate css file.